docs/semgrep-code/findings.md

```diff
@@ -52,6 +52,20 @@ To view findings individually, toggle **Group by Rule** to **No grouping** using
 
 ![Group by Rule option](/img/cloud-platform-findings-no-grouping.png#md-width)
 
+### Default Code page view
+
+:::tip
+If you do not see any findings after a scan has concluded, check the **Projects** page to view the findings count, if any, and to set a [primary branch](/deployment/primary-branch), if it is not already set.
+:::
+
+The Semgrep Code page displays findings from all projects (repositories) with a [primary branch](/deployment/primary-branch), arranged by most recent scan.
```

Comment:
> arranged by most recent scan. 

For this bit, are you referring to which instance across branches do we pick to show? If so, then we are only picking the one on the primary branch on first page load. 

Or if you're referring to the actual order that the findings are listed within each rule group, they're ordered by when the finding was last opened (by most recent to least recent).

Reply:
It's this one:
> For this bit, are you referring to which instance across branches do we pick to show? If so, then we are only picking the one on the primary branch on first page load.

I will clarify!